packages/vue/src/components/search/DataSearch.jsx

```diff
@@ -262,6 +278,27 @@ const DataSearch = {
 			this.setValue(this.$props.defaultValue, true);
 		}
 	},
+	mounted() {
+		document.addEventListener('keydown', this.onKeyDown);
+		const { focusShortcuts } = this.$props;
+		// dynamically import hotkey-js
+		if (!isEmpty(focusShortcuts)) {
+			this.hotKeyCombinationsUsed = isHotkeyCombinationUsed(focusShortcuts);
+
+			if (this.hotKeyCombinationsUsed) {
```

Comment:
@mohdashraf010897 You don't need to import it dynamically. Import at the top. Fix it in `react` too. If `hot-keys` can handle the single keys then just use that and remove your custom logic to keep the bundle size minimum.